spec/controllers/network_router_controller_spec.rb

```diff
@@ -123,7 +128,12 @@
           :priority    => MiqQueue::HIGH_PRIORITY,
           :role        => 'ems_operations',
           :zone        => @ems.my_zone,
-          :args        => [{:name => "test"}]
+          :args        => [{
+            :name            => 'test',
+            :admin_state_up  => 'true',
+            :ems_id          => "@ems.id",
```

Comment:
Test failure for Ruby is related to this line. Do you really want to have `"@ems.id"` or `@ems.id` as a string. If latter you should use `@ems.id.to_s`. Travis has to be green :)